packages/wrangler/src/dev.tsx

```diff
@@ -805,12 +780,11 @@ export async function startDev(args: StartDevOptions) {
 					enableServiceEnvironments: !(args.legacyEnv ?? true),
 				},
 				experimental: {
-					// only pass `experimentalAssetsOptions` if it came from args not from config
+					// only pass `assetsOptions` if it came from args not from config
 					// otherwise config at startup ends up overriding future config changes in the
 					// ConfigController
-					assets: args.experimentalAssets
-						? experimentalAssetsOptions
-						: undefined,
+					// TODO @Carmen should `assets` still be under `experimental` here?
```

Comment:
does `assets` need to stay under `experimental` here? // cc @RamIdeas

Reply:
If it's no longer experimental then no need 👍